### DISCUSSION TITLE
Will this unit work FDXS35F2VEB

### DISCUSSION THREAD

**Discussion by jdvorecky** [revk/xESP32-Faikout#204] [Q&A]
Hi, i have two concealed Daiking FDXS35F2VEB  which should work with Intesis INWMPDAI001R000 will work via the P1/P2 connectors. I read below that P1/P2 is not possible without extra hardware as its serial communication.
But i have looked at the daikin schematics and board and it does have a connector X50A and X35A as well.
so i wonder if the Faikin will work for this type of AC unit.
Any ideas?   
thanks for your help.
![image](https://github.com/revk/ESP32-Faikin/assets/93602120/e676aab8-98b9-4a5f-97d0-52850a9e8317)
![image](https://github.com/revk/ESP32-Faikin/assets/93602120/773eed3a-caa6-49f4-85cd-b52b83c99eee)

**Comment by revk** [comment]
If it has X50A and X35A you are probably in luck, indeed. Looks like mine.

**Comment by jdvorecky** [comment]
Perfect 👌. I'll give it a shot.Do you by any chance know any other place i can get the faikin board than from UK. Im in Slovakia so wonder if maybe i can buy it online from Germany or Austria? Or is Amazon uk the only place?Thanks again for the help. 
-------- Original message --------From: RevK ***@***.***> Date: 1/26/24  16:35  (GMT+01:00) To: revk/ESP32-Faikin ***@***.***> Cc: jdvorecky ***@***.***>, Author ***@***.***> Subject: Re: [revk/ESP32-Faikin] Will this unit work FDXS35F2VEB (Discussion #204) 
If it has X50A and X35A you are probably in luck, indeed. Looks like mine.

**Reply by revk** [reply]
We have them on Amazon and that covers quite a few places. Sorry if that is a pain, some people had the UK leave the EU and made some shit harder.

**Comment by jdvorecky** [comment]
Oh yeah, so do i just use the x50 port or do i need x35 for anything?Can you maybe send me a picture how you hooked it up exactly? Which pin to which? Thanks 
-------- Original message --------From: RevK ***@***.***> Date: 1/26/24  16:35  (GMT+01:00) To: revk/ESP32-Faikin ***@***.***> Cc: jdvorecky ***@***.***>, Author ***@***.***> Subject: Re: [revk/ESP32-Faikin] Will this unit work FDXS35F2VEB (Discussion #204) 
If it has X50A and X35A you are probably in luck, indeed. Looks like mine.

**Comment by jdvorecky** [comment]
Hi Revk, can you tell me if for my unit all i need is the port X50  or do i need the X35 for something as well?
on the board it seems like there are two X50 ports, one is in top left of the picture (i know you cant see it from this angle but its looks like its X50)  already hooked up to something, and then in the middle there is another X50 port thats open. do you think they are connected in parallel somehow? should i use the open one or try to connect to the existing wires on the top left X50 port?
if it works on yours, can you maybe share the wiring scheme ?
thanks for your help.

**Reply by revk** [reply]
The X35 is normally needed to provide the power and the X50 for data lines. Mine had wires to both (I am sure there are pictures on here somewhere). No idea in two X50 ports - I can't imagine any reason they would be in parallel to may be fine to use the second one, but cannot say for sure. Be careful working inside any electrical equipment.

**Reply by njh** [reply]
Some photos here:
https://github.com/revk/ESP32-Faikin/issues/23#issuecomment-1488616649
